### ISSUE TITLE
Error: Interactive not found in package ceramic

### ISSUE THREAD

**Issue by AlbertoEAF** [ceramic/ceramic#57]
Hello, 

I was trying out ceramic on my Ubuntu 18.04, I got to say the website looks amazing!

Unfortunately the sample code on the main page fails:

![Screenshot from 2019-05-10 23-26-49](https://user-images.githubusercontent.com/3976753/57559748-4ce8af00-737b-11e9-80e0-889a441e06be.png)

However, the code in http://ceramic.github.io/docs/tutorial.html works for me and the hello world app shows up.

Thank you for this amazing project!
Kind regards,
Alberto

**Comment by ghost** [comment]
Same as here.

**Comment by vindarel** [comment]
It seems ok: https://github.com/ceramic/ceramic/issues/38

> the new version of the API has dropped the interactive function. I updated the documentation, but not the code on the main project page.

**Comment by ghost** [comment]
Ah I see. Thanks @vindarel

**Comment by stacksmith** [comment]
@vindarel: does this mean that 'interactive' may be safely ignored?

**Comment by ghost** [comment]
@stacksmith I think yes.

Also, I've opened a PR to remove the deprecated function from the main project page - https://github.com/ceramic/ceramic.github.io/pull/3

**Comment by ghost** [comment]
I think this issue can be closed at this point since interactive has been removed both from the doc and codebase?

**Comment by AlbertoEAF** [comment]
Let me try again @momozor :)

**Comment by AlbertoEAF** [comment]
Thanks, now it works :)
